parsers/quarto.c

```diff
@@ -0,0 +1,135 @@
+/*
+ *
+ *  Copyright (c) 2022, Masatake YAMATO
```

Comment:
As far as quickly reading, this parser is mostly the same as Rmarkdown parser.
Do we need a new parser for Quarto? If yes, why do you think so?

Reply:
I think we do for the following reasons:

1. The QMD quarto "chunks" use slightly different pattern for labeling that RMD files do which may affect how tags are extracted
1. Although similar, to be able to use tag-bar style plugins (e.g. as in vim), would need the qmd extension to be supported.
1. The QMD style seems to still be evolving with varying language support, so likely may start diverging more and more from RMD

As a heavy RMD user that has switched to Quarto recently, I personally would enjoy being able to easily extract tags with minimal increased overhead (essentially took the __Rmarkdown__ code and revised it slightly)